package.yaml

```diff
@@ -134,3 +134,7 @@ tests:
     - directory
     - temporary
     - extra
+    - containers
+    - QuickCheck
+    - spago
+    - extra
```

Comment:
```suggestion
```

Reply:
I don't think this can be removed, it's being used for `nubOrd`

Reply:
It's already being imported in the line preceding your additions

Reply:
Oh of course, haha :man_facepalming: